package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "root",
-    "lockfileVersion": 2,
+    "lockfileVersion": 3,
```

Comment:
Q: what's the cause of this?

Reply:
I was using npm version 9 which sets this to 3, to allow backwards compatibility.  I downgraded to npm v8.  The readme only lists a version number for node.  Perhaps we want to fill out the rest of the tool versioning?  Maybe in the configuration.md?

Reply:
Yeah good idea. We should set an npm version and document it. Please open a ticket for that